docs/maps/map-settings.asciidoc

```diff
@@ -0,0 +1,38 @@
+[role="xpack"]
+[[maps-settings]]
+== Map settings
+
+To configure map settings, click *Map settings* in the application toolbar.
+
+[float]
+[[maps-settings-navigation]]
+=== Navigation
+
+*Zoom range*::
+Constrain the map to the defined zoom range.
+
+*Initial map location*::
+Configure the initial map center and zoom.
```

Comment:
I'm wondering if you can move this sentence so it works as an intro sentence under the Navigation title.  So its similar to what you did in the Spatial filters section.

Reply:
Not sure this is a good intro sentence to navigation. There are some future configuration values that will require a broader introduction sentence.